### PR TITLE
fix/ ecpay dockerfile

### PR DESCRIPTION
**Pull request by ryo29wx**
## Overview
- Fix each github action's workflow files.
- FIx to be able to build docker files.
- FIx to be able to build each go files.

## Verification
can build dockerfile and gofiles on each directory with local macbook.

### CODE REVIEW

**Review by yabamuro — COMMENTED** [review]
Almost fine with me to approve but please let me ask you just one question.
I understood that you changed the way to build go resources in accordance with go versions' upgrade.
How come 'xxxx.git' is the output binary file?
Is it the new go's specification?
If so, do you think xxx.git should be fine name convention or do you think we should name the other way to it?

**Comment by ryo29wx** [comment]
It must be specified in the go module file. The .git suffix must be specified by you, but you may want to modify it.

File: ecfront/Dockerfile [hunk 1]
```diff
@@ -51,13 +51,13 @@ WORKDIR /apl/
 COPY templates ./templates
 COPY assets ./assets
 RUN mkdir -p /mnt/gcs/$GCS_BUCKET_NAME
-COPY --from=builder /go/src/ecfront/main /apl/main
```
Comment (ryo29wx) [inline ecfront/Dockerfile:54]: rename a binary file to ecfront

File: ecfront/go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module github.com/mockten/mockten/ecfront.git
```
Comment (ryo29wx) [inline ecfront/go.mod:1]: delete `.git` suffix from  module name

**Review by yabamuro — APPROVED** [review]

**Pull request merged**